data/remnant/remnant ships.txt

```diff
@@ -893,47 +893,32 @@ ship "Merganser"
 		licenses
 			Remnant
 		"cost" 6830000
```

Comment:
whilst I dont approve of most of these changes, if we go with them we probably need to bring the cost down